package.json

```diff
@@ -364,7 +364,7 @@
     "lint-prettier": "yarn && yarn lint-prettier-pure",
     "lint-prettier-pure": "prettier --check '{frontend,enterprise/frontend,e2e}/**/*.{js,jsx,ts,tsx,css}'",
     "lint-yaml": "yamllint **/*.{yaml,yml} --ignore=node_modules/**/*.{yaml,yml}",
-    "precommit": "lint-staged",
+    "precommit": "lint-staged --shell",
```

Comment:
This is needed to make it pick up the ENV, as pointed out by @WiNloSt 
The alternative is [what lint-staged suggest which is using cross-env](https://github.com/lint-staged/lint-staged#use-environment-variables-with-linting-commands), but we don't have it as a dependency at the moment